src/structs/jpeg_header.rs

```diff
@@ -360,7 +360,7 @@ impl JPegHeader {
                     let rval = usize::from(rbits(segment[hpos], 4));
                     if lval >= 2 || rval >= 4
                     {
-                        return err_exit_code(ExitCode::UnsupportedJpeg,"DQT has invalid index");
+                        break;
```

Comment:
I am less sure about this change, but that's the logic in C++, so I'm reproing it here as well.